docs/kb/rules/ruleset-default-mode.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Adjust the default mode for a whole ruleset.
+tags:
+  - Rules
+  - Semgrep Registry
+  - Semgrep Code
+append_help_link: true
+---
+
+# Why do new rules keep appearing in Comment or Block mode?
+
+Semgrep AppSec Platform [Policies](/docs/semgrep-code/policies) can contain both individual rules and **rulesets**, which are curated groups of rules recommended for particular purposes. All organizations in the AppSec Platform start with two rulesets: the `default` ruleset and the `comment` ruleset. These rulesets are curated to be good starter packs for security teams to review, and for developers to see comments about, respectively.
+
+As Semgrep adds new rules to improve coverage, some of these rules are also added to rulesets. If you have a ruleset added to your organization's policies, any new rules will be added to the ruleset automatically, and become part of your policies also.
+
+The `default` and `comment` rulesets are initially added in Monitor mode, which is primarily intended for security teams to review. You can also [add new rulesets to your policies](/docs/semgrep-code/policies#add-rulesets-to-your-policies-from-the-registry) from the Semgrep Registry.
```

Comment:
```suggestion
The `default` and `comment` rulesets are initially added in **Monitor** mode, where the findings generated by the rules are primarily intended for security teams to review. You can also [add new rulesets to your policies](/docs/semgrep-code/policies#add-rulesets-to-your-policies-from-the-registry) from the Semgrep Registry.
```
I don't know if I captured the gist of what you're saying, but I think I'm trying to clarify whether the security team is reviewing the rules themselves or the findings they generate